lib/matplotlib/tests/test_sankey.py

```diff
@@ -67,7 +67,7 @@ def test_sankey2():
     s = Sankey(flows=[0.25, -0.25, 0.5, -0.5], labels=['Foo'],
                orientations=[-1], unit='Bar')
     sf = s.finish()
-    assert np.all(np.equal(np.array((0.25, -0.25, 0.5, -0.5)), sf[0].flows))
+    assert np.array_equal(sf[0].flows, [0.25, -0.25, 0.5, -0.5])
```

Comment:
Better to use numpy.testing.assert_array_equal() because that gives better error messages. I wouldn’t bother to go through all tests and change it, but if you rewrite something, that’s the way to go.